orchagent/aclorch.cpp

```diff
@@ -952,43 +952,7 @@ bool AclTable::create()
 
     sai_attribute_t attr;
     vector<sai_attribute_t> table_attrs;
-
-    int32_t range_types_list[] =
-        { SAI_ACL_RANGE_TYPE_L4_DST_PORT_RANGE,
-          SAI_ACL_RANGE_TYPE_L4_SRC_PORT_RANGE
-        };
-
-    set<sai_acl_bind_point_type_t> binds;
-    for (const auto& portid_pair : ports)
-    {
-        Port port;
-        if (!gPortsOrch->getPort(portid_pair.first, port))
-        {
-         continue;
-        }
-
-        switch (port.m_type)
-        {
-        case Port::PHY:
-            binds.insert(SAI_ACL_BIND_POINT_TYPE_PORT);
-            break;
-        case Port::VLAN:
-            binds.insert(SAI_ACL_BIND_POINT_TYPE_VLAN);
-            break;
-        case Port::LAG:
-            binds.insert(SAI_ACL_BIND_POINT_TYPE_LAG);
-            break;
-        default:
-            return SAI_STATUS_FAILURE;
-        }
-    }
-
-    vector<int32_t> bpoint_list;
-    for (auto bind : binds)
-    {
-        bpoint_list.push_back(bind);
-    }
-
+    vector<int32_t> bpoint_list = { SAI_ACL_BIND_POINT_TYPE_PORT, SAI_ACL_BIND_POINT_TYPE_LAG };
```

Comment:
Also add SAI_ACL_BIND_POINT_TYPE_VLAN?

Reply:
yes, can add this, then we don't need to change the code in the future when deciding to support VLAN.